packages/epo-react-lib/package.json

```diff
@@ -85,8 +86,14 @@
   },
   "peerDependencies": {
     "i18next": "^22.0.4",
+    "next": "^13.0.5",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-i18next": "^12.0.0"
+  },
+  "peerDependenciesMeta": {
+    "next": {
```

Comment:
add Next.js as an optional peer, components with no reliance on it will be fine but those that use its `Link` or `Image` components will need the peer.